### PR TITLE
Fixup pulp_label migration

### PR DESCRIPTION
**Pull request by mdellweg**
Added a repair command and fixed the dependency issue.

[noissue]

### CODE REVIEW

File: pulpcore/app/migrations/0098_pulp_labels.py [hunk 1]
```diff
@@ -53,5 +84,5 @@ class Migration(migrations.Migration):
             name='pulp_labels',
             field=django.contrib.postgres.fields.hstore.HStoreField(default=dict),
         ),
-        migrations.RunPython(code=copy_labels_up, reverse_code=migrations.RunPython.noop, elidable=True),
+        migrations.RunPython(code=copy_labels_up, elidable=True),
```
Comment (gerrod3) [inline pulpcore/app/migrations/0098_pulp_labels.py:87]: Why has the noop been dropped?

File: pulpcore/app/management/commands/datarepair-labels.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                        Label.objects.filter(content_type=ctype, object_id=OuterRef("pulp_id"))
+                        .annotate(label_data=RawSQL("hstore(array_agg(key), array_agg(value))", []))
+                        .values("label_data")
+                    )
+                    model.objects.update(pulp_labels=label_subq)
+                    Label.objects.filter(content_type=ctype).delete()
+
+        if Label.objects.count() and purge and not dry_run:
+            print("Purge remaining labels.")
+            Label.objects.all().delete()
```
Comment (lubosmj) [inline pulpcore/app/management/commands/datarepair-labels.py:77]: This deletes all the objects, is it expected? What is its real use?

File: pulpcore/app/migrations/0098_pulp_labels.py (hunk 1 above)
Comment (mdellweg) [reply pulpcore/app/migrations/0098_pulp_labels.py:87]: It shouldn't have been there i used it for debugging.

File: pulpcore/app/management/commands/datarepair-labels.py (hunk 2 above)
Comment (mdellweg) [reply pulpcore/app/management/commands/datarepair-labels.py:77]: Only if you put `--purge` on the cmdline intentionally.

I expect that we remove the `Label` model in 3.25, but only if it's empty. We need to give admins a way to unblock themselves.

**Review by lubosmj — APPROVED** [review]

**Review by gerrod3 — APPROVED** [review]

**Pull request merged**